Day-3/Instructions.md

```diff
@@ -0,0 +1,16 @@
+# Day 3
+Create a function that take in an argument and print the argument to the screen on click of the button
+
+## Detailed explanation
+- In the `Day-3/` directory, create `index.html`, create `index.js`
+- In the JS file, create a function that takes in a parameter
+- In the HTML file, create a button that calls the function `onclick`, and pass an argument of string to the function.
+- Whatever is been passed within the button function, should be printed to the screen.
```

Comment:
Whatever is passed into the function from the onclick call on the button should be printed to screen.

HINTS:
- Use document.write() or console.log() to print to the page
- Do you print inside your function